src/main/java/io/jenkins/plugins/zscaler/ZscalerScan.java

```diff
@@ -123,6 +123,7 @@ BuildDetails getBuildDetails(Run build, TaskListener listener) throws IOExceptio
     buildDetails.setBuildTriggeredBy(env.get("USER"));
     buildDetails.addEventDetails("build_url", env.get("BUILD_URL"));
     buildDetails.addEventDetails("action", build.getDisplayName());
```

Comment:
Lets change this as well.
We need to send build_number separately instead of key as "action"